app/src/main/java/com/keylesspalace/tusky/fragment/TimelineFragment.java

```diff
@@ -432,24 +432,26 @@ private void showNothing() {
     public void onActivityCreated(@Nullable Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
 
-        if (jumpToTopAllowed()) {
-            TabLayout layout = requireActivity().findViewById(R.id.tab_layout);
-            if (layout != null) {
-                onTabSelectedListener = new TabLayout.OnTabSelectedListener() {
-                    @Override
-                    public void onTabSelected(TabLayout.Tab tab) {
-                    }
+        if (requireActivity() instanceof TabbedActivity) {
+            if (jumpToTopAllowed()) {
```

Comment:
`jumpToTopAllowed()` should no longer be needed - if the Activity is a `TabbedActivity` we can always jump to the top. Also it is outdated - hashtag tabs can now be included in the main tabs.

Reply:
1. Remove jumpToTopAllowed;
2. Move add/remove tab listener to the setMenuVisibility method;
3. Added a listener at the conversation fragment.